app/components/forms/wizard/basic-details-step.js

```diff
@@ -141,7 +141,14 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     $.fn.form.settings.rules.checkMaxMinOrder = () => {
       return parseInt($('.ui.form').form('get value', 'ticket_min_order'), 10) <= parseInt($('.ui.form').form('get value', 'ticket_max_order'), 10);
     };
-
+    window.$.fn.form.settings.rules.checkValidTimeDifference = () => {
+      const STA = document.getElementsByName('start_time')[0].value.split(':');
+      const STIS = (parseInt(STA[0]) + parseInt(STA[1])) * 60;
+      const ETA = document.getElementsByName('end_time')[0].value.split(':');
+      const ETIS = (parseInt(ETA[0]) + parseInt(ETA[1])) * 60;
+      return STIS < ETIS;
```

Comment:
there should be an easy way for comparing times using moment.js . And please use varilable names as they are used rest of place.

Reply:
@maze-runnar is it ok now?